changelog.d/cli-254.added

```diff
@@ -0,0 +1 @@
+Support fail-open in CI: adds --suppress-errors/--no-suppress-errors (defaults to disabled)
```

Comment:
What does "disabled" mean in this context?

Reply:
"disabled" in this context means that errors are not suppressed by default. Rephrasing to 
```
Support fail-open in CI: adds --suppress-errors/--no-suppress-errors (defaults to --no-suppress-errors)
```